pkg/api/v1/atlascustomresource.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 //+k8s:deepcopy-gen=false
+
```

Comment:
[q] does linter require a new line between annotations and normal comments?

Reply:
It looks like kubebuilder still recognizes these comments, while godoc doesn't, so our docs aren't polluted with tags.

I'm actually considering changing this in Cluster as well, although it will bloat the file somewhat.

Reply:
Answering your question: yes, the linter enforces a specific godoc comment format - it has to start with the name of whatever you're commenting, followed by its description. In this case the first thing was this k8s comment which did not pass the check.